src/PactNet.Abstractions/IRequestBuilder.cs

```diff
@@ -204,7 +204,14 @@ public interface IRequestBuilderV3
         /// <param name="body">Request body</param>
         /// <param name="contentType">Content type</param>
         /// <returns>Fluent builder</returns>
-        IRequestBuilderV3 WithBody(string body, string contentType);
+        IRequestBuilderV3 WithBody(string body, string contentType);
+
+        /// <summary>
+        /// A Multipart body containing a single part, which is an uploaded file
+        /// </summary>
+        /// <param name="body">Request body</param>
+        /// <returns>Fluent builder</returns>
+        IRequestBuilderV3 WithMultipartSingleFileUpload(string filePath);
```

Comment:
@uglyog is this supported on the V2 interface as well? I'm assuming it is, because there is nothing inherently new about the spec it writes out?

Reply:
It uses the content type matcher, which V3.

Reply:
ok cool, we'll leave it out then. Thanks.